### ISSUE TITLE
ADB in recovery need some time to start,

### ISSUE THREAD

**Issue by User-green** [ubports/ubports-installer#2497]
... witch is not mentioned in the instruction, made me reboot device into recovery 3+ times

**UBports Installer `0.8.9-beta` (snap)**
Environment: `ManjaroLinux 20 Qonos linux 5.10.102-1-MANJARO x64 NodeJS v12.18.3`
Device: jasmine_sprout
Target OS: Ubuntu Touch
Settings: `undefined`
OPEN-CUTS run: *N/A*
Log: *N/A*

I cant be shure, but it seem that this is device-specific and will pop up at any install.

**Comment by NeoTheThird** [comment]
where exactly in the installation process does this cause issues? what did you see? a connection lost popup?

**Comment by User-green** [comment]
> 

At the point of using ADB. Installer asked me to press some physical buttons on the device to start ADB. After doing this I saw recovery poping up, but installer were saying, that it (installer) is still waiting for me to press buttons and ADB to start. 

I was a bit confused at this point, beacuse I have never saw recovery with ADB before. After searching thought recovery for a while and finding no "Enable ADB" (or simular) option I restarted the device for a cople of times because I thought that I did something wrong. After third restart into the recovery I walked away for a couple of minutes to google the issue. When I returned, I found out, that the installer for whatever reason was saying, that ADB connection is now established. I supose that the installer needs some time to found the device, but this is said nowhere.
